mlflow/projects/databricks.py

```diff
@@ -229,8 +232,6 @@ def jobs_runs_get(self, databricks_run_id):
 
 
 def _get_tracking_uri_for_run():
-    if not tracking.utils.is_tracking_uri_set():
```

Comment:
We remove the old behavior here, which attempted to default the tracking URI to "databricks" if it was not set. Unfortunately, this logic was called only after we'd already created a local run, so the Databricks Job would attempt to fetch a run with the UUID of the local run from the Databricks tracking server.